src/content/getting-started/cloud-provider-accounts/vmware-vsphere/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+linkTitle: Preparing VMware vSphere for CAPI
+title: Preparing VMware vSphere for Cluster API
+description: How to set up your VMware vSphere Cluster tenant to run Giant Swarm management clusters and workload clusters under your jurisdiction.
+weight: 80
+user_questions:
+  - What are the VMware vSphere prerequisites for use with Giant Swarm?
+  - What do I need to configure in VMware vSphere in order to run Giant Swarm clusters?
+  - What user permissions do I need to run Giant Swarm clusters in VMware vSphere?
+menu:
+  main:
+    identifier: gettingstarted-infraprovider-vsphere
+    parent: gettingstarted-infraprovider
+last_review_date: 2023-10-03
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rocket
+---
+In order to run the Giant Swarm platform in your VMware vSphere environment, a number of prerequisites must be satisfied to support Cluster API Provider vSphere (CAPV).
+
+## vSphere infrastructure
+
+It is recommended to create resource pool for your cluster(s). However, it is possible to deploy them in the root resource pool (a.k.a. vSphere cluster).
+
+To use PersistentVolumes (PV), the cluster needs support for Cloud Native Storage (CNS), which is available in vSphere 6.7 Update 3 and later.
+
+## vSphere User permissions
+
+The user role assigned to the CAPV user must have at least the following permissions:
+
+| Categorie | permissions |
+| -------- | -------- |
+| Datastore | `Allocate space`<br>`Browse datastore`<br>`Low level file operations` |
+| global | `Disable methods`<br>`Enable methods`<br>`Licenses` |
+| Network | `Assign network` |
+| Resource | `Assign virtual machine to resource pool` |
+| Sessions | `Message`<br>`Validate session` |
+| Profile-driven storage | `Profile-driven storage view` |
+| vApp | `Import` |
+| Virtual machine | `Configuration/Change Configuration`<br>`Configuration/Add existing disk`<br>`Configuration/Add new disk`<br>`Configuration/Add or remove device`<br>`Configuration/Advanced configuration`<br>`Configuration/Change CPU count`<br>`Configuration/Change Memory`<br>`Configuration/Change Settings`<br>`Configuration/Configure Raw device`<br>`Configuration/Extend virtual disk`<br>`Configuration/Modify device settings`<br>`Configuration/Remove disk`<br>`Configuration/Create from existing`<br>`Configuration/Remove`<br>`Interaction/Power offPower off`<br>`Interaction/Power on`<br>`Provisioning/Deploy template` |
+
+The role must be assigned to the following objects:
+
+* vCenter Server
+* Datacenters or datacenter folders
+* Hosts and  clusters
+* VM templates
+* Resource pools  (With Propagate to children)
+* Distributed Port Group
+* Distributed Switch
+* VM and Template folders (With Propagate to children).
+
+### Considerations for single cluster failure domains
+
+If you want to leverage failure domains at the host level where a groupe of hosts is a failure domain (stretched cluster, racks, PDU distribution...), CAPV will need to work with Anti-Affinity rules.
```

Comment:
I don't recall if we tested failure domains or not. Do you recall?

Reply:
We haven't tested it no